src/DIRAC/ResourceStatusSystem/PolicySystem/Actions/BaseAction.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, name, decisionParams, enforcementResult, singlePolicyResults,
         self.enforcementResult = enforcementResult
         self.singlePolicyResults = singlePolicyResults
         self.clients = clients
-        self.log = gLogger.getSubLogger(self.__class__.__name__)
+        self.log = gLogger.getSubLogger(__name__)
```

Comment:
why do you do the opposite of what you do everywhere ?

Reply:
sorry, must have been tired